auth/middleware.js

```diff
@@ -0,0 +1,30 @@
+const { User } = require("../models");
+const { toData } = require("./jwt");
+
+function auth(req, res, next) {
+  const auth =
```

Comment:
by @akellbl4:
I don't use express because I prefer koa. But as I see in documentation recommended way to get a header from a request is req.get("%header_name") or req.header("%header_name").
But I think it is not a criminal to get it in this way.